### ISSUE TITLE
Put kubectl commands into a shared function

### ISSUE THREAD

**Issue by rancher-max** [rancher/qa-tasks#967]
Right now there are a lot of kubectl commands scattered throughout. It would be nice to have one shared function that runs all kubectl commands that can just take in a couple args to simplify some of the code and rework.

Related to Epic https://github.com/rancher/qa-tasks/issues/799

**Comment by fmoral2** [comment]
https://github.com/rancher/distros-test-framework/blob/2ff8ae8ad52acf8d6fb3347f4b6bc57a59b1ada8/shared/cluster.go#L139C1-L139C5

**Comment by rancher-max** [comment]
yeah we have it, just don't use it across the board yet and I think should

**Comment by fmoral2** [comment]
totally , i just added as reference.
